src/request/swap-iframe/index.js

```diff
@@ -0,0 +1,9 @@
+/* global runKeyguard */
+/* global SwapIFrameApi */
+
+runKeyguard(SwapIFrameApi, {
+    loadNimiq: false,
```

Comment:
I think it's always required?
So could also be true.

Reply:
Not if we use this iframe for other swaps in the future as well, e.g EUR-BTC.